pkg/metadata/scheduler.go

```diff
@@ -32,10 +32,9 @@ type Scheduler struct {
 }
 
 // NewScheduler builds and returns a new Metadata Scheduler
-func NewScheduler(s *serializer.Serializer, hostname string) *Scheduler {
+func NewScheduler(s *serializer.Serializer) *Scheduler {
 	scheduler := &Scheduler{
-		srl:      s,
-		hostname: hostname,
+		srl: s,
 	}
```

Comment:
What do you think about removing `hostname` field from `Scheduler` type?

Reply:
Fixed :)